src/app/shared/constants/flags/spell_dbc_school_options.ts

```diff
@@ -1,11 +0,0 @@
-import { Flag } from '../../types/general';
-
-export const SPELL_DBC_SCHOOL_OPTIONS: Flag[] = [
-  { bit: 0, name: 'Physical' },
-  { bit: 1, name: 'Holy' },
-  { bit: 2, name: 'Fire' },
-  { bit: 3, name: 'Nature' },
-  { bit: 4, name: 'Frost' },
-  { bit: 5, name: 'Shadow' },
-  { bit: 6, name: 'Arcane' },
-];
```

Comment:
why?

Reply:
I removed it because `spell-school-mask.ts` had the same values already  (which get used in item-template) so that there is no code duplication.